client.go

```diff
@@ -154,7 +187,7 @@ func (c *SpiceClient) createClient(address string, systemCertPool *x509.CertPool
 				"RetryableStatusCodes": [ "UNAVAILABLE", "UNKNOWN", "INTERNAL" ]
 	        }
 	    }]
-	}`, c.maxRetries)
+	}`, c.maxRetries+1)
```

Comment:
is this going to ensure it can be set to zero to disable?

Reply:
Yes, maxAttempts here includes the initial request.

https://github.com/grpc/grpc-proto/blob/master/grpc/service_config/service_config.proto#L137